static/scss/partials/landing/a.scss

```diff
@@ -61,3 +77,363 @@
+            background: white;
+            align-items: center;
+            border-radius: $border-radius-md;
+            text-align: center;
+            padding: $spacing-xl;
+
+            .c-price {
+                color: $color-blue-50;
+                font-weight: 500;
+                font-size: 1.125em;
+                margin: 0;
+                padding-top: $spacing-md;
+
+                .c-currency {
+                    color: $color-dark-gray-50;
+                    font-size: .7em;
+                }
+
+                .premium-price {
+                    font-size: 1.75em;
+                }
+            }
+
+            .c-features {
+                padding-top: $spacing-md;
+                text-align: left;
+
+                li {
+                    padding-bottom: $spacing-md;
+                }
+
+                li::before {
+                    content: "";
+                    width: 20px;
+                    height: 20px;
+                    margin-top: 6px;
+                    margin-left: -$spacing-lg;
+                    position: absolute;
+                    background-repeat: no-repeat;
+                    background-image: url("/static/images/icon-blue-check.svg");
+                }
+            }
+
+            .c-description {
+                font-size: .875em;
+                border-radius: $border-radius-md;
+                border: 2px solid $color-light-gray-40;
+                padding: $spacing-sm $spacing-md;
+            }
+
+            .mzp-c-button {
+                margin-top: $spacing-lg;
+                float: bottom;
+            }
+        }
+
+    }
+
+    .c-pricing-text {
+        grid-row-start: 1;
+        grid-column-start: span 3;
+        height: 100%;
+        text-align: center;
+
+        h2 {
+            font-family: $font-metropolis;
+            font-weight: 500;
+            font-size: 2.5em;
+        }
+
+        .c-text {
+            padding: 0 $spacing-lg;
+        }
+
+        @media screen and ($mq-lg) {
+            text-align: start;
+            grid-column-start: span 2;
+            grid-row-start: auto;
+
+            .c-text {
+                padding: 0;
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                height: 100%;
+            }
+        }
+    }
+}
+
+.use-case-section {
+    box-shadow: 0px 15px 15px $color-light-gray-20;
+}
+
+.c-use-case-content {
+    display: grid;
+    grid-template-columns: 1fr;
+    row-gap: $spacing-md;
+   
+    .use-case-title {
+        padding: $spacing-md $spacing-md $spacing-sm $spacing-md;
+        display: flex;
+        font-family: $font-metropolis;
+        border-radius: $border-radius-md;
+    }
+
+    .use-case-title-text {
+        color: $color-black;
+        font-weight: bold;
+        border-radius: $border-radius-md;
+        font-family: $font-metropolis;
+        font-size: 1.125em;
```

Comment:
Use the [typography sizing](https://protocol.mozilla.org/fundamentals/typography.html#code) from Protocol to set the size.